### DISCUSSION TITLE
Custom IDE editor

### DISCUSSION THREAD

**Discussion by NullR3F** [studio-minus/ppg-bugs-and-requests#2048] [Ideas]
Make a code editor specialized to create ppl playground mods with it's own api.

**Comment by dinevolence** [comment]
How do you expect zooi to make this?

**Comment by mestiez** [comment]
It is more likely that a Visual Studio Code addon would be created for this purpose. However, this is already possible just by referencing the correct assemblies in Visual Studio [as described here](https://www.studiominus.nl/ppg-modding/intro/boilerplate.html).
